editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -2283,11 +2289,11 @@ void Node3DEditorViewport::_sinput(const Ref<InputEvent> &p_event) {
 			} else if (unicode == '-') {
```

Comment:
You can use `KEY_MINUS` and `KEY_KP_MINUS` here (non-physical), and remove `unicode`.